src/components/home/CharacterList.vue

```diff
@@ -0,0 +1,69 @@
+<script>
+export default {
+  data: () => ({
+    characterList: [
+      {
+        name: "Thomas Shelby",
+        description: "Thomas Michael ‘Tommy’ Shelby OBE DCM MM MP is the main protagonist of the Peaky Blinders",
+        picture: "https://focus.independent.ie/thumbor/4Gb2trRUxYLirgM1fmq6Dv8WiJk=/203x33:1755x889/960x640/prod-mh-ireland/c2e70d08-c450-11ed-aa5d-0210609a3fe2.png"
+      },
+      {
+        name: "Grace Shelby",
+        description: "Grace Helen Shelby (née Burgess) is an Irish woman who takes up employment as a barmaid at The Garrison Pub, and secretly operates as an undercover agent for the Birmingham Police",
+        picture: "https://metro.co.uk/wp-content/uploads/2020/06/PRI_154740850.jpg?quality=90&strip=all"
+      },
+      {
+        name: "Arthur Shelby",
+        description: "Arthur Shelby Jr. is one of the main protagonists of the Peaky Blinders. He is the eldest son",
+        picture: "https://hips.hearstapps.com/hmg-prod/images/arthur-shelby-in-peaky-blinders-season-6-1646316779.jpg?crop=0.646xw:0.431xh;0.228xw,0.137xh&resize=1200:*"
+      },
+      {
+        name: "John Shelby",
+        description: "John Michael Shelby (also known as Johnny and John Boy) is one of the two former tritagonists (alongside Polly Gray) of Peaky Blinders",
+        picture: "https://www.nme.com/wp-content/uploads/2020/04/joe-cole-peaky-blinders-696x442.jpg"
+      },
+      {
+        name: "Polly Gray",
+        description: "Elizabeth Pollyanna \"Polly\" Gray (née Shelby) is one of the two former tritagonists",
+        picture: "https://i.pinimg.com/originals/bb/e8/00/bbe800a9ed0719b918086700f383acbe.png"
+      }
+    ],
```

Comment:
in normal cases this data comes from backend and it make sense move mock data in separate constant file (and all text)